packages/nimble-components/src/label-provider/core/label-token-defaults.ts

```diff
@@ -10,5 +10,6 @@ export const coreLabelDefaults: { readonly [key in TokenName]: string } = {
     popupIconWarningLabel: 'Warning',
     popupIconInformationLabel: 'Information',
     filterSearchLabel: 'Search',
-    filterNoResultsLabel: 'No items found'
+    filterNoResultsLabel: 'No items found',
+    loadingLabel: 'Loading'
```

Comment:
@fredvisser? @jattasNI? I'm indifferent.

Reply:
Indifferent, other than if it is used we use the single character unicode ellipses instead of three periods.

```suggestion
    loadingLabel: 'Loading…'
```

Reply:
I'd vote to include the ellipsis.

I did a very brief search for guidance in other design systems and found that [Adobe Spectrum suggests](https://spectrum.adobe.com/page/grammar-and-mechanics/#Ellipsis) to "use an ellipsis at the end of the text to communicate the progress of something that is in process, such as a waiting or loading state." So at least we won't be the only ones taking this approach.